src/main.less

```diff
@@ -1,11 +1,11 @@
 @import url('@educandu/educandu/styles/main.less');
 
-@global-primary-color: brown;
+@global-primary-color: @pink-6;
```

Comment:
It's even more pink now. Prepare for an eye burn!

Reply:
ouch!

Reply:
This is weirdly not so bad :D

Reply:
We can also change the colors, I don't care much either way so far, just wanted to make the links visible. And make sure we don't forget what website we're on when we run OMA 😋